README.md

```diff
@@ -41,8 +41,11 @@ least one paver. Use `S` as the start state, and `r`, `g`, `b` as terminals that
 represent the tiles.
 
 ## A1.1
-
-«replace this with your answer»
+```
+S -> r S r | r  
+S -> g S g | g  
+S -> b S b | b  
+```
 
```

Comment:
# 5

I really like that form of expressing it. I'd always used:

~~~
       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b
~~~

but I think yours is clearer.